blog/views.py

```diff
@@ -7,21 +7,16 @@
 
 from blog.models import Blog, Comment, Profile, UserVote
 from blog.permissions import IsOwnerOrReadOnly
-from blog.serializers import (
-    BlogSerializer,
-    CommentSerializer,
-    ProfileSerializer,
-    UserSerializer,
-    VoteSerializer
-)
+from blog.serializers import (BlogSerializer, CommentSerializer,
```

Comment:
use isort for automatically sorting the imports.

Reply:
this one is actually sorted by isort.